hooks/tk-nuke_actions.py

```diff
@@ -160,7 +160,7 @@ def _import_script(self, path, sg_publish_data):
         if not os.path.exists(path):
             raise Exception("File not found on disk - '%s'" % path)
 
-        nuke.nodePaste(path)
+        nuke.nodePaste(path.encode('utf-8'))
```

Comment:
I am missing a comment here and it seems this is the wrong way around again.
Am I right thinking that we are turning a unicode path into `utf-8` encoded string of bytes?
Why is path a unicode string? I think the issue we should fix is upstream - like i have pointed out in the past, we want things to be utf-8 for as long as possible in toolkit and if they need to be converted to unicode objects, we do that at the "bondary" of toolkit, right before it leaves the system and enters another. 

On line 138 i see this line:

```
path = self.get_publish_path(sg_publish_data).replace(os.path.sep, "/").decode("utf-8")
```

So we are decoding it, only to encode it a few lines later down in the code? This doesn't look right to me.

Reply:
In all other DCCs, we are decoding path at this level. I noticed that at some places that come afterwards we still need utf-8 encoded strings, but we can go the other way.

Reply:
why can't you just remove the `.decode("utf-8")` suffix on line 138 and then you don't need any of the encoding in `_create_read_node`, `_import_script` and `_open_project`?

Reply:
yes, I was testing that, it works, but we still need to place a decode in import_script